mars/services/task/supervisor/processor.py

```diff
@@ -191,7 +189,12 @@ async def _process_stage_chunk_graph(
                     meta_api.get_chunk_meta.delay(c.key, fields=["bands"])
                 )
                 fetch_op_keys.append(c.op.key)
-        key_to_bands = await meta_api.get_chunk_meta.batch(*get_meta_tasks)
+        # TODO(fyrestone): A more general way to get the key to bands
+        # for all execution backends.
+        try:
+            key_to_bands = await meta_api.get_chunk_meta.batch(*get_meta_tasks)
+        except KeyError:
```

Comment:
When will raise `KeyError`?

Reply:
When executed by the Ray execution backend.

Reply:
For Ray, it won't report to meta service after execution?

Reply:
Currently, Ray execution backend only updates the task result meta & lifecycle. It updates the stage meta to the graph for iterative tiling.

Reply:
Before finding a better solution for meta part, better to add comments here.

Reply:
There is a TODO comment here, is that OK?

Reply:
Fine.